src/pages/PassportRedemption/style.tsx

```diff
@@ -12,23 +12,24 @@ type PassportProps = {
   mainView?: boolean;
 };
 
-const PassportContainer = styled.div`
+const CardContainer = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
   width: 367px;
   margin: 0 auto;
   background: #ffffff;
-  
-  box-shadow: 0 0 10px rgba(0,0,0,0.25);
+
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.25);
   border: 1px solid #dedede;
   border-radius: 20px;
   box-sizing: border-box;
   overflow: hidden;
 
   position: absolute;
+  // opacity: ${(props: PassportProps) => (props.mainView ? '1' : '0.85')};
```

Comment:
meant to comment out?

Reply:
yeah for now, if we change the opacity, the background bleeds through the card, so for  now ill comment it out, but i think we can incorporate it again in the future